### ISSUE TITLE
Warning is printed twice

### ISSUE THREAD

**Issue by linsui** [federico-terzi/espanso#85]
I got this message.

```
WARNING: using legacy espanso config location in $HOME/.espanso is DEPRECATED
Starting from espanso v0.3.0, espanso config location is changed.
Please check out the documentation to find out more: https://espanso.org/docs/configuration/
WARNING: using legacy espanso config location in $HOME/.espanso is DEPRECATED
Starting from espanso v0.3.0, espanso config location is changed.
Please check out the documentation to find out more: https://espanso.org/docs/configuration/
```

**Comment by federico-terzi** [comment]
Hi @linsui,
Thanks for the feedback. I will try to solve the bug in the next release. 
Regards

**Comment by Boxfort** [comment]
I've proposed a fix for this in PR #87
